helm/zot/values.schema.json

```diff
@@ -333,6 +333,9 @@
                 "additionalLabels": {
                     "type": "object"
                 },
+                "basicAuth": {
+                    "type": "object"
+                },
```

Comment:
Don't you have to add the properties here?

Reply:
Until we have a tool that can generate it, I guess no